README.md

```diff
@@ -45,6 +45,7 @@ This is a list of examples and scripts compiled by the Snyk Customer Experience
 - [How to generate a list (CSV) of all user and service accounts in a group](userlist)
 - [Pysnyk examples](https://github.com/snyk-labs/pysnyk/tree/master/examples)
 - [Tool for bulk project deletion](bulk-delete)
+- [How to extract all issues pertinent to a tagged project](tagged-project-issues)
```

Comment:
lets move this to the top since its newest!

Reply:
Done.